fmn/consumer/producer.py

```diff
@@ -55,7 +55,7 @@ def work(self):
         for pref in fmn.lib.models.Preference.list_batching(self.session):
 
             if not pref.context.name in self.backends:
-                return
+                continue
```

Comment:
Because with the current code it silently crashes (stops), because it doesn't ack the next message.
This is the case where we are unable to send a specific type of mesasge, so let's just ignore it for now.